src/libraries/System.Reflection/tests/ParameterInfoTests.cs

```diff
@@ -83,7 +83,16 @@ public void HasDefaultValue_broken_on_NETFX(Type type, string name, int index, b
         public void HasDefaultValue_ReturnParam()
         {
             ParameterInfo parameterInfo = GetMethod(typeof(ParameterInfoMetadata), "Method1").ReturnParameter;
-            Assert.True(parameterInfo.HasDefaultValue);
```

Comment:
Even though the issue was not about changing the return parameter default value changing this to false makes sense and do not think it could be breaking change, are OK with this change @steveharter?

Reply:
Since return values don't have a default value, one would expect `false` to be returned. This is a separate bug than the original, but fixing it here is fine IMO.

I don't think it rises to the level of a breaking change, however. Most likely, users will never be inspecting `MethodInfo.ReturnParameter.HasDefaultValue`. This is particularly broken today since a method that returns `bool`, for example, has `HasDefaultValue==true` but `DefaultValue` is `null` which is not even a valid value for `bool`.